source/release-notes/v1.4-release-notes.rst

```diff
@@ -0,0 +1,136 @@
+    sudo yum clean all
+    sudo yum update ondemand
+
+    # Optionally remove dependencies from prior versions of OOD
+    sudo yum remove rh-nodejs010\* rh-passenger40\* rh-ruby22\* nginx16\*
+
+
+.. warning::
+
+  As always please update the *development* or *test* instances of OnDemand installed at your center first before you modify the *production* instance.
+
+.. warning::
+
+   As a security enhancement application development privileges must now be enabled on a per user basis. TODO add link.
+
+Infrastructure Version Changes
+------------------------------
+
+OnDemand's infrastructure components have been merged into a monolithic repository. Component changelogs have been frozen and the parent repository will now track all infrastructure changes: `OnDemand 1.4.9 <https://github.com/OSC/ondemand/blob/v1.4.9/CHANGELOG.md>`__. `Diff with 1.3.7 <https://github.com/OSC/ondemand/compare/v1.3.7...v1.4.9>`__
+
+
+Application Version Changes
+----------------------------
+
+.. _v1.4-app-versions:
+.. list-table:: Application Versions
+   :widths: auto
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - App
+     - Version
+   * - `Dashboard App`
+     - 1.26.2 → `1.30.2 <https://github.com/OSC/ood-dashboard/blob/v1.30.2/CHANGELOG.md>`__
+       (`diff <https://github.com/OSC/ood-dashboard/compare/v1.26.2...v1.30.2>`__)
+   * - `Shell App`
+     - 1.3.1 → `1.4.2 <https://github.com/OSC/ood-shell/blob/v1.4.2/CHANGELOG.md>`__
+       (`diff <https://github.com/OSC/ood-shell/compare/v1.3.1...v1.4.2>`__)
+   * - `Files App`
+     - 1.4.1 → `1.5.0 <https://github.com/OSC/ood-fileexplorer/blob/v1.5.0/CHANGELOG.md>`__
+       (`diff <https://github.com/OSC/ood-fileexplorer/compare/v1.4.1...v1.5.0>`__)
+   * - `File Editor App`
+     - 1.3.3 → `1.4.0 <https://github.com/OSC/ood-fileeditor/blob/v1.4.0/CHANGELOG.md>`__
+       (`diff <https://github.com/OSC/ood-fileeditor/compare/v1.3.3...v1.4.0>`__)
+   * - `Active Jobs App`
+     - 1.6.2 → `1.6.8 <https://github.com/OSC/ood-activejobs/blob/v1.6.8/CHANGELOG.md>`__
+       (`diff <https://github.com/OSC/ood-activejobs/compare/v1.6.2...v1.6.8>`__)
+   * - `Job Composer App`
+     - 2.8.3 → `2.10.1 <https://github.com/OSC/ood-myjobs/blob/v2.10.1/CHANGELOG.md>`__
+       (`diff <https://github.com/OSC/ood-myjobs/compare/v2.8.3...v2.10.1>`__)
+   * - `Desktops App`
+     - Unchanged
+
+:numref:`v1.4-app-versions` lists the versions as well as the previous version
+it was updated from for each of the system web applications in this release.
+
+
+Details
+-------
+
+Upgrade to Ruby 2.4, NodeJS 6, Passenger 5
+............................................
+
+This upgrade updates our dependencies to Software Collections Ruby 2.4 and NodeJS 6. Passenger is also seeing an upgrade to version 5, but until Passenger 5 is supported by SCL OSC will host the Passenger 5 and NGINX 1.14 RPMs which are built based on "@efranz where did we get the Passenger RPM build information from?".
```

Comment:
We use this repo for the RPM builds: https://github.com/phusion/passenger_rpm_automation.